src/OAuth2/ResponseType/JwtAccessToken.php

```diff
@@ -127,5 +127,15 @@ protected function createPayload($client_id, $user_id, $scope = null)
             'token_type' => $this->config['token_type'],
             'scope'      => $scope
         );
+        
+        if (isset($this->config['jwt_extra_payload']) && is_callable($this->config['jwt_extra_payload'])) {
+            $func = $this->config['jwt_extra_payload'];
+            $extra = $func($client_id, $user_id, $scope);
+            if (is_array($extra)) {
+                $payload = array_merge($extra, $payload);
```

Comment:
no, I man instead of:
```php
$extra = call_user_func($this->config['jwt_extra_payload'], $client_id, $user_id, $scope);
if (is_array($extra)) {
    $payload = array_merge($extra, $payload);
}
```
do this:
```php
$payload = call_user_func($this->config['jwt_extra_payload'], $payload);
```